### ISSUE TITLE
Variance update in tf.contrib.layer.batch_norm

### ISSUE THREAD

**Issue by mkabra** [tensorflow/tensorflow#13064]
Hi,

In the current implementation of batch norm, the variance to be used for update to moving variance is computed by centering the data around the current mean. The more proper way would be to compute the variance around the moving mean. With the current method, the moving variance will depend upon the batch size. With the updated method, the batch size will have less influence on the moving variance.

Thanks,
Mayank

**Comment by cy89** [comment]
@sguada this seems like a reasonable observation. Should we ask @mkabra for a PR?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
A member of the TensorFlow organization has replied after the stat:awaiting tensorflower label was applied.

**Comment by sguada** [comment]
Sure let see a PR and evaluate it.

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by rthadur** [comment]
Contrib folder is deprecated and moved to tf addons , will be closing this request. Thank you
